### PR TITLE
fixed applications style and menu

### PR DESCRIPTION
**Pull request by tiana25**

### CODE REVIEW

File: src/app/shell/personal-cabinet/applications/applications.component.html [hunk 1]
```diff
@@ -66,7 +66,8 @@
 
 
         <ng-template #ChildApplication>
-          <h3> {{ child.firstName }} {{ child.lastName }} {{ child.middleName }}</h3>
+          <h3 class='name'> {{ child.firstName }} {{ child.lastName }} {{ child.middleName }} ({{ child.dateOfBirth | date
+            :'shortDate'}})</h3>
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/applications/applications.component.html:70]: date: constants.SHORT_DATE_FORMAT

File: src/app/shell/personal-cabinet/workshops/workshops.component.html [hunk 2]
```diff
@@ -47,7 +47,7 @@
   <ng-container *ngIf="applications?.length else isEmptyList;">
     <div *ngFor="let child of (childrenCards$ | async)?.entities">
       <div *ngIf="isApplications(applications, child)">
-        <h3> {{ child.firstName }} {{ child.lastName }} {{ child.middleName }} ({{ child.dateOfBirth | date
+        <h3 class='name'> {{ child.firstName }} {{ child.lastName }} {{ child.middleName }} ({{ child.dateOfBirth | date
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/workshops/workshops.component.html:50]: double quotes

File: src/app/shell/personal-cabinet/applications/application-card/application-card.component.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <mat-card *ngIf="application" fxLayout='row' fxLayoutAlign='space-evenly center'
   [ngClass]="application.status === applicationStatus.Pending ? 'card card-new' : 'card'">
   <mat-card-content class="card-block" fxLayout='column' fxLayoutAlign='start start'>
-    <div fxLayout='column' fxLayoutAlign='start start'>
+    <div class='block1' fxLayout='column' fxLayoutAlign='start start'>
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/applications/application-card/application-card.component.html:4]: double qoutes

File: src/app/shell/personal-cabinet/applications/application-card/application-card.component.html [hunk 4]
```diff
@@ -13,23 +13,25 @@ <h4 class="name title">{{
   </mat-card-content>
 
   <mat-card-content class="card-block" fxLayout='column' fxLayoutAlign='start center'>
-    <div fxLayout='column' fxLayoutAlign='start start'>
+    <div class='block2' fxLayout='column' fxLayoutAlign='start start'>
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/applications/application-card/application-card.component.html:16]: double qoutes

File: src/app/shell/personal-cabinet/applications/application-card/application-card.component.html [hunk 5]
```diff
@@ -13,23 +13,25 @@ <h4 class="name title">{{
...
       <h4 class="title text-link" [routerLink]="['/workshop-details', application?.workshopId ]">{{
         application?.workshop.title }}</h4>
-
-      <p class="text" *ngIf="userRole === role.parent">
-        <mat-icon class="text_icon">home</mat-icon>
-        {{application?.workshop.providerTitle}}
-      </p>
-
+        <div class="organization" fxLayout='row' fxLayoutAlign='center center' *ngIf="userRole === role.parent">
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/applications/application-card/application-card.component.html:19]: double qoutes

File: src/app/shell/personal-cabinet/applications/application-card/application-card.component.html [hunk 6]
```diff
@@ -13,23 +13,25 @@ <h4 class="name title">{{
...
+          <mat-icon class="text_icon">home</mat-icon>
+          <p class="text">{{application?.workshop.providerTitle}}</p>
+        </div>
+        <div class="workshop-age" fxLayout='row' fxLayoutAlign='center center' *ngIf="userRole === role.parent">
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/applications/application-card/application-card.component.html:23]: double qoutes

File: src/app/shell/personal-cabinet/applications/application-card/application-card.component.html [hunk 7]
```diff
@@ -13,23 +13,25 @@ <h4 class="name title">{{
...
+          <mat-icon class="text_icon">person</mat-icon>
+          <p class="text">{{application?.workshop.minAge}} - {{application?.workshop.maxAge}}</p>
+        </div>
     </div>
   </mat-card-content>
 
-  <mat-card-content class="card-block" fxLayout='column' fxLayoutAlign='start start'>
-    <div *ngIf="application" [matMenuTriggerFor]="menu" #trigger="matMenuTrigger" (mouseenter)="trigger.openMenu()"
+  <mat-card-content class="card-block" fxLayout='column' fxLayoutAlign='start center'>
+    <div class='block3' *ngIf="application" [matMenuTriggerFor]="menu" #trigger="matMenuTrigger" (mouseenter)="trigger.openMenu()"
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/applications/application-card/application-card.component.html:31]: double qoutes

File: src/app/shell/personal-cabinet/applications/application-card/application-card.component.html [hunk 8]
```diff
@@ -13,23 +13,25 @@ <h4 class="name title">{{
...
+        <div class="workshop-age" fxLayout='row' fxLayoutAlign='center center' *ngIf="userRole === role.parent">
+          <mat-icon class="text_icon">person</mat-icon>
+          <p class="text">{{application?.workshop.minAge}} - {{application?.workshop.maxAge}}</p>
+        </div>
     </div>
   </mat-card-content>
 
-  <mat-card-content class="card-block" fxLayout='column' fxLayoutAlign='start start'>
-    <div *ngIf="application" [matMenuTriggerFor]="menu" #trigger="matMenuTrigger" (mouseenter)="trigger.openMenu()"
+  <mat-card-content class="card-block" fxLayout='column' fxLayoutAlign='start center'>
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/applications/application-card/application-card.component.html:30]: double qoutes

**Review by litvinets — CHANGES_REQUESTED** [review]

File: src/app/shell/personal-cabinet/applications/application-card/application-card.component.html [hunk 9]
```diff
@@ -1,7 +1,7 @@
-<mat-card *ngIf="application" fxLayout='row' fxLayoutAlign='space-evenly center'
+<mat-card *ngIf="application" fxLayout="row" fxLayoutAlign="space-evenly center"
   [ngClass]="application.status === applicationStatus.Pending ? 'card card-new' : 'card'">
-  <mat-card-content class="card-block" fxLayout='column' fxLayoutAlign='start start'>
-    <div fxLayout='column' fxLayoutAlign='start start'>
+  <mat-card-content class="card-block" fxLayout="column" fxLayoutAlign="start start">
+    <div class="block1" fxLayout="column" fxLayoutAlign="start start">
```
Comment (dmitryy90) [inline src/app/shell/personal-cabinet/applications/application-card/application-card.component.html:4]: please use meaningful class names

File: src/app/shell/personal-cabinet/applications/application-card/application-card.component.scss [hunk 10]
```diff
@@ -91,3 +105,23 @@
...
+.material-icons {
+  font-size: 16px;
+  line-height: 14.6px;
+}
+.organization {
+  padding-bottom: 8px;
+  color: #3849F9;
+}
+.block2 {
+  width: 120px;
```
Comment (dmitryy90) [inline src/app/shell/personal-cabinet/applications/application-card/application-card.component.scss:120]: did you try relative units? percents?

File: src/app/shell/personal-cabinet/applications/applications.component.html [hunk 11]
```diff
@@ -66,7 +66,7 @@
 
 
         <ng-template #ChildApplication>
-          <h3> {{ child.firstName }} {{ child.lastName }} {{ child.middleName }}</h3>
+          <h3 class="name"> {{ child.firstName }} {{ child.lastName }} {{ child.middleName }} ({{ child.dateOfBirth | date: constants.SHORT_DATE_FORMAT }})</h3>
```
Comment (dmitryy90) [inline src/app/shell/personal-cabinet/applications/applications.component.html:69]: please use meaningful class names. child-name

File: src/app/shell/personal-cabinet/personal-cabinet.component.scss [hunk 12]
```diff
@@ -2,26 +2,41 @@
...
+    opacity: 1;
+  }
 }
 .active {
-  color: #3849f9;
+  color: #3849f9!important;
   opacity: 1;
   border-bottom: 2px solid #3849f9;
 }
 ::ng-deep .mat-tab-link {
```
Comment (dmitryy90) [inline src/app/shell/personal-cabinet/personal-cabinet.component.scss:35]: potential risk of overriding styles in other components

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

**Review by dmitryy90 — APPROVED** [review]

**Review by litvinets — APPROVED** [review]

**Pull request merged**